core/includes/common.inc

```diff
@@ -3048,6 +3048,19 @@ function backdrop_deliver_html_page($page_callback_result) {
     backdrop_add_http_header('X-Frame-Options', $frame_options);
   }
 
+  if ($site_config->get('block_interest_cohort', TRUE)) {
```

Comment:
`config->get()` only accepts one parameter (unlike `variable_get`) so you'll need to delete `, TRUE`.